examples/basic/poptext/poptext.py

```diff
@@ -0,0 +1,47 @@
+import movis as mv
+import json
+
+scene = mv.layer.Composition(size=(1080, 1900), duration=4.0)
+scene.add_layer(
+    mv.layer.Image("background.png"),
+)
+
+scene.add_layer(
+    mv.layer.Audio("output.ogg"),
```

Comment:
In my environment, "output.ogg" was not found. Can you check this?

Reply:
output.ogg is generated by GCP's Text-to-Speech. I could add the generated output.ogg but it's ~200K bin.
I've added the GCP feature.

Reply:
Thanks!